src/presentation/controllers/ingest-controller.ts

```diff
@@ -0,0 +1,24 @@
+import { BaseController, PostRequest, RestController } from './base-controller'
+import { HttpErrorHandler } from '../interfaces/http-error-handler'
+import { Exception } from '../../model/exceptions/exception'
+import { Request, Response } from 'express'
+import { IngestService } from '../../business-logic/services/interfaces/ingest-service'
+
+@RestController('/ingest-operations')
```

Comment:
Is the ingest-operations needed? Won't it suffice with the url `/rundowns/:rundownId/reingest`?
And if so, is it necessary with a ingest controller?

Reply:
Good suggestion, this now resides in the `RundownController` instead.